server/datastore/mysql/migrations/tables/20210818151828_AddJSONKeyValueTable.go

```diff
@@ -90,13 +90,13 @@ func Up_20210818151828(tx *sql.Tx) error {
 		&config.HostExpirySettings.HostExpiryEnabled,
 		&config.HostExpirySettings.HostExpiryWindow,
 		&config.ServerSettings.LiveQueryDisabled,
-		&config.HostSettings.AdditionalQueries,
+		&config.Features.AdditionalQueries,
```

Comment:
Hmmm not sure we should ever change migrations? I understand this one is because it did not compile... I thought we had copied the AppConfig setting inside the migration (I believe Michal mentioned doing something like that, maybe we didn't do it in the end)... Anyway, I can't really see that being a problem, but since the non-legacy config will likely change again in the future, I wonder if we should bite the bullet now and just duplicate the old AppConfig struct as a migration-local type immediately, so we don't run into more issues with it.

Reply:
thank you! makes a lot of sense. I hardcoded the config in the test in https://github.com/fleetdm/fleet/pull/7358/commits/e95d6d66500239d4464ed218c44e922ffcbc21dd